exporter/loadbalancingexporter/loadbalancer.go

```diff
@@ -86,6 +99,15 @@ func newLoadBalancer(params exporter.CreateSettings, cfg component.Config, facto
 		}
 	}
 
+	if oCfg.Resolver.AWSCloudMap != nil {
+		awsCloudMapLogger := params.Logger.With(zap.String("resolver", "awsCloudMap"))
+		var err error
+		res, err = newCloudMapResolver(awsCloudMapLogger, &oCfg.Resolver.AWSCloudMap.NamespaceName, &oCfg.Resolver.AWSCloudMap.ServiceName, oCfg.Resolver.AWSCloudMap.Port, &oCfg.Resolver.AWSCloudMap.HealthStatus, oCfg.Resolver.AWSCloudMap.Interval, oCfg.Resolver.AWSCloudMap.Timeout)
```

Comment:
For more than 3 parameters, I think I would prefer a new struct with the resolver settings (or reuse the one from the config?)